src/v2/templates/people/person-list.php

```diff
@@ -201,62 +208,35 @@ function emptyOrUnassignedJSON($stuff): string
 
         // setup datatables
         'use strict';
-        var bVisible = parseInt("<?= SystemConfig::getValue('bHidePersonAddress') ? 0 : 1 ?>");
-
         let dataTableConfig = {
             deferRender: true,
             search: { regex: true },
             columns: [
                 {
                     title:i18next.t('Actions'),
                 },
-                {
-                    title:i18next.t('Id'),
-                    visible:false
-                },
-                {
-                    title:i18next.t('Last Name'),
-                },
-                {
-                    title:i18next.t('First Name'),
-                },
-                {
-                    title:i18next.t('Address'),
-                    visible:bVisible
-                },
-                {
-                    title:i18next.t('Home Phone'),
-                },
-                {
-                    title:i18next.t('Cell Phone'),
-                },
-                {
-                    title:i18next.t('Email'),
-                },
-                {
-                    title:i18next.t('Gender'),
-                },
-                {
-                    title:i18next.t('Classification'),
-                },
-                {
-                    title:i18next.t('Roles'),
-                },
-                {
-                    title:i18next.t('Properties'),
-                    visible:false
-                },
-                {
-                    title:i18next.t('Custom'),
-                    visible:false
-                },
-                {
-                    title:i18next.t('Group'),
-                    visible:false
+                <?php
+                $firstVisibleColumnId = PHP_INT_MAX;
+                $columnId = 0;
+                $columnIdMap = array();
+                $columns = json_decode(SystemConfig::getValue('sPersonListColumns'), null, 512, JSON_THROW_ON_ERROR);
+                foreach ($columns as $column) {
+                    $columnId++;
+                    $columnIdMap[$column->name] = $columnId;
+                    $columnTitle = array('title' => 'i18next.t(\'' . $column->name . '\')');
```

Comment:
if you want to make this a bit simpler
```suggestion
                    $columnTitle = array('title' => "i18next.t('{$column->name}')"
```

Reply:
Good. Done.